gradle/libs.versions.toml

```
@@ -80,6 +81,7 @@ firebase-crashlytics-ktx = { module = "com.google.firebase:firebase-crashlytics-
 firebase-config-ktx = { module = "com.google.firebase:firebase-config-ktx", version.ref = "firebase-config" }
 timber = { module = "com.jakewharton.timber:timber", version.ref = "timber" }
 androidx-core-splashscreen = { group = "androidx.core", name = "core-splashscreen", version.ref = "splashscreen" }
+kakao = { module = "com.kakao.sdk:v2-user", version.ref = "kakao"}
```

카카오는 group, name, version.ref로 나누는게 좋지 않을까!? kakao link도 들어갈거니까?!
```suggestion
kakao = { group = "com.kakao.sdk", name =":v2-user", version.ref = "kakao"}
```

---

반영완료! commit suggestion 누르니까 자동으로 resolve가 되어버리네.. 확인 부탁함~